pkg/systemd/services/service-details.jsx

```diff
@@ -319,6 +332,53 @@ export class ServiceDetails extends React.Component {
                 });
     }
 
+    deleteAction() {
+        this.getUnitPaths().then(unitPaths => {
+            this.setState({ showDeleteDialog: true, unitPaths });
+        });
+    }
+
+    async getUnitPaths() {
+        const paths = [this.props.unit.FragmentPath];
+
+        await Promise.all(this.props.unit.Triggers.map(async trigger => {
+            // Getting dbus properties from a non-loaded unit is not possible so resort to systemctl show
+            const unitPath = await cockpit.spawn(["systemctl", "show", "--value",
+                "--property", "FragmentPath", trigger]);
```

Comment:
Does an error here actually trigger the `.catch()` below? That seems to happen in a different scope. Can you try to intentionally break this command (locally) and make sure it doesn't cause a page oops due to an unhandled promise rejection or (more likely) unhandled exception? As this is `await`ed, it seems it would need a `catch() {} ` block.

Reply:
It did get catched:

```
failed to look up unit details: systemctl exited with code 1
```